library/CMakeLists.txt

```diff
@@ -157,10 +157,26 @@ if(GEN_FILES)
             ${CMAKE_CURRENT_SOURCE_DIR}/../scripts/generate_ssl_debug_helpers.py
             ${error_headers}
     )
+
+    add_custom_command(
```

Comment:
I don't know what's wrong, but all out-of-source builds are failing. To reproduce:
```
rm -rf mbedtls_out_of_source_build; MAKEFLAGS=' ' ./tests/scripts/all.sh -k test_cmake_out_of_source
```